src/bot.js

```diff
@@ -254,6 +254,23 @@ client.on(Events.MessageCreate, async message => {
 		let userInput = message.content
 			.replace(new RegExp("^\s*" + myMention.source, ""), "").trim();
 
+		// Process text files if attached
+		if (message.attachments.size > 0) {
+			const textAttachments = Array.from(message.attachments, ([, value]) => value).filter(att => att.contentType.startsWith("text"));
+			if (textAttachments.length > 0) {
+				userInput = `${message.content}`;
```

Comment:
userInput should be appended to, not overriden
also this code should be done after checking for commands and formatting etc, [on this line](https://github.com/mekb-turtle/discord-ai-bot/pull/20/files#diff-eccd86fca21bf18f16c33e56d78a057ed364743326438601731484529bf4a058R354)